Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -107,178 +129,195 @@ static void EFB_Write(u32 data, u32 addr)
 	}
 }
 
+u32 dbat_table[1 << (32 - 17)];
+u32 ibat_table[1 << (32 - 17)];
```

Comment:
It probably doesn't matter much where we put the definition.  Yes, 17 should probably be a define.